.github/workflows/release_repository.yaml

```diff
@@ -6,15 +6,12 @@ on:
     paths:
       - "infra/repository/**"
 
-env:
-  DIR: "infra/repository"
-
 jobs:
-  code_review:
+  release:
     uses: ./.github/workflows/call_release_infra.yaml
```

Comment:
infra or repository?

Reply:
small fix, don't know if it does worth to open another pr

Reply:
Probably it does because of some issue I found today..